dashboard/app/controllers/courses_controller.rb

```diff
@@ -107,6 +107,34 @@ def edit
     render 'edit', locals: {unit_group: unit_group}
   end
 
+  def vocab
+    unit_group = UnitGroup.get_from_cache(params[:course_name])
+    raise ActiveRecord::RecordNotFound unless unit_group
+    # Assumes if one unit in a unit group is migrated they all are
+    return render :forbidden unless unit_group.scripts_for_user(current_user)[0].is_migrated
```

Comment:
`scripts_for_user` will hit the database more times than necessary. this should accomplish the same result for cheaper:
```suggestion
    return render :forbidden unless unit_group.default_scripts[0].is_migrated
```

Reply:
in the long run, all of our scripts (e.g. csd1-2020) will be migrated, but we will still not want to have a vocab page for them, because their lesson plans will still be on CB. in that future, I am thinking that choice will either be something like 
1. `unit_group.default_scripts[0].lessons.first.has_lesson_plan` (or whatever fields we need to look at)
2. `unit_group.has_lesson_plans` (or another new field on the unit group)

see later comments to track this work rather than doing it now.